### PR TITLE
Add AWS ELB + Route53 Configuration

### PR DESCRIPTION
**Pull request by frodenas**

### CODE REVIEW

File: aws.md [hunk 1]
```diff
@@ -2576,11 +2576,87 @@ meta:
...
-      pass: (( vault meta.vault_prefix "/rds:password" ))
+      pass: (( vault "secret/aws/staging/rds:password" ))
     uaadb:
       host: "xxxxxx.rds.amazonaws.com" # <- your RDS Cluster endpoint
       user: "admin"
-      pass: (( vault meta.vault_prefix "/rds:password" ))
+      pass: (( vault "secret/aws/staging/rds:password" ))
+```
+
+Not it's time to create our Elastic Load Balancer that will be in front of the `gorouters`, but as we will need TLS termination we then need to create a SSL/TLS certificate for our domain.
```
Comment (drnic) [inline aws.md]: /Not/Now/

**Comment by drnic** [comment]
Sexy

**Comment by xiujiao** [comment]
When I `make deploy` to apply elb changes, it errors out as 
` aws_iam_server_certificate.staging-cf-elb-cert: [WARN] Error uploading server certificate, error: AccessDenied: User: arn:aws:iam::169146138861:user/bosh is not authorized to perform: iam:UploadServerCertificate on resource: arn:aws:iam::169146138861:server-certificate/xj-new-vpc-staging-cf-elb-cert4m45jffxrza53b3a2umubl3ysy`

What I did is I went to AWS Console, and assign `IAMFullAccess` to my `bosh` user, then I tried `make deploy` again, the elb stuff is successfully applied. To tight down permission, after elb changes are applied, I detached `IAMFullAccess` from my `bosh` user.

@frodenas Did you see the same problem?

**Comment by frodenas** [comment]
@xiujiao You need to add to your user a custom policy, this has been documented recently, look for `ServerCertificates`: https://github.com/starkandwayne/codex/blob/master/aws.md#generate-access-key

**Comment by xiujiao** [comment]
@frodenas Thank you! I followed elb branch for the elb part without looking back at the beginning of the doc. That is a nice change!

**Comment by xiujiao** [comment]
@frodenas What do you think we add security group `open_vpn` Geoff created to our elb? I could not `cf api` on my bastion for staging (it times out) until I added `open_vpn` to the security group for elb.

**Comment by xiujiao** [comment]
@frodenas  After talk with Geoff, `open_vpn` allows 443 from everywhere, we may need narrow it down in future. For now, it is better than `wide_open`

**Comment by frodenas** [comment]
@xiujiao Why not fix the `dmz` security group opening ports `80`, `443` and `4443`?

**Comment by xiujiao** [comment]
@frodenas from the Inbound and outbound rules from`dmz`, it maybe a good idea we just create another security group called `cf-elb`, in `cf-elb`, we enable 80, 443 and 4443

**Pull request merged**